### DISCUSSION TITLE
Dedicated IP for apps

### DISCUSSION THREAD

**Discussion by pierratdev** [caprover/caprover#1383] [Ideas]
Hello, I'm New user of CapRover (great job the team !!!).

I think it would be interesting to be able to visually can add/assign a dedicated IP address for an app if desired, so I propose the idea.

Big thank !!!

**Comment by c43721** [comment]
I'm not sure that would work because you only get 1 dedicated IP for the VPS. How would this improve your current experience?

**Comment by oskar-gmerek** [comment]
@c43721 In the case you need more IP addresses there is no problem to buy one or few for that VPS.
A feature like this would be good for separating services. 

@pierratdev I'm not pretty sure, but I think that should be possible by editing Nginx Config
